parlai/core/logs.py

```diff
@@ -234,6 +246,17 @@ def log_final(self, setting, report):
         }
         for key, value in report.items():
             self.run.summary[key] = value
+        if self.model_file is not None and self.log_model:
+            artifact = self._wandb.Artifact('model', type='model')
```

Comment:
Is it possible to pass a unique name for the model instead of just `model`? I think something can be extracted from `model_file`.

Reply:
Fixed this in the latest commit that adds model checkpoints and tags for model.